docs/src/modules/spring/pages/spring-boot-integration.adoc

```diff
@@ -0,0 +1,32 @@
+= Spring Boot Integration
+
+The Kalix Spring SDK provides a familiar developer experience for Java developers that are used to Spring. Its `Main`
+class is annotated with `@SpringBootApplication` and starts exactly the same way as in a Spring Boot application, namely
```

Comment:
Both seem correct, below one sounds better to me but might be just a matter of taste, so feel free to ignore.

```suggestion
class is annotated with `@SpringBootApplication` and starts exactly the same way as a common Spring Boot application, namely
```